lessons/pydata/pandas_groupby/index.ipynb

```diff
@@ -0,0 +1,3297 @@
+{
```

Comment:
<ul><li>Ani jeden z argumentů <code>dtype</code> , <code>usecols</code> a <code>parse_dates</code>  jsme si asi neukazovali, nebo jo, Kubo? Asi by to chtělo zmínit a možná znovu dát link na <code>read_csv</code></li><li><code>list(dtypes)</code>  mi přijde mírně neintuitivní - buď bych to vypsal nebo aspoň okomentoval</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>A potřebujem vlastně categoricals? Nebude to tady jedno? Je to malinký dataset a akorát to bude lidi plést.</p>

Reply:
<p>Vyhodil jsem categorials, tim padem i list(dtypes), dtype atd. usecols vysvětlim na hodině</p>